opencti-platform/opencti-front/src/utils/graph/LassoSelection.tsx

```diff
@@ -79,7 +79,7 @@ const LassoSelection: FunctionComponent<LassoSelectionProps> = ({
     coord = reposition(e);
     currentContext.moveTo(coord.x, coord.y);
     currentContext.lineWidth = 1;
-    currentContext.setLineDash([0.5, 1]);
+    currentContext.setLineDash([1, 3]);
```

Comment:
Why did you change that line ? (curious)

Reply:
@Kedae is the real author of this

Reply:
Because I love css :) It's juste adjustment